src/main/kotlin/kweb/Kweb.kt

```diff
@@ -53,8 +57,7 @@ private val CLIENT_STATE_TIMEOUT: Duration = Duration.ofHours(48)
 
 class Kweb private constructor(
         val debug: Boolean,
-        val plugins: List<KwebPlugin>,
-        val buildPage: WebBrowser.() -> Unit
```

Comment:
This is the breaking change I mentioned in the PR and should probably be deprecated instead